ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3IIOPResourceDefinition.java

```diff
@@ -57,12 +57,14 @@
             new SimpleAttributeDefinitionBuilder(EJB3SubsystemModel.USE_QUALIFIED_NAME, ModelType.BOOLEAN, true)
                     .setAllowExpression(true)
                     .setFlags(AttributeAccess.Flag.RESTART_NONE)
+                    .setRequired(true)
```

Comment:
Please remove the 'true' from L57 as that says the opposite of this setRequired(true).